docs/monitoring-usage.rst

```diff
@@ -278,3 +278,91 @@ follows::
 
 .. _Time Series:
     https://cloud.google.com/monitoring/api/ref_v3/rest/v3/TimeSeries
+
+
+Writing Custom Metrics
+---------------------------
+
+The Stackdriver Monitoring API can be used to write data points to custom metrics. Please refer to
+the documentation on `Custom Metrics`_ for more information.
+
+To write a data point to a custom metric, you must provide an instance of
+:class:`~gcloud.monitoring.metric.Metric` created with a metric type as well as values for
+the labels associated with that metric type. The metric type can either
+be created using the :class:`~gcloud.monitoring.metric.MetricDescriptor` class or be
+auto-created.  For more information on creating metric descriptors, see the `Metric Descriptors`_
+section. For information on metric type auto-creation, see `Auto-creation of custom metrics`_.
+
+You must also create a fully parameterized :class:`~gcloud.monitoring.resource.Resource` from one
+of the available resource descriptors which can be listed using the
+:class:`~gcloud.monitoring.resource.ResourceDescriptor` class, which similarly requires that its
```

Comment:
- You don't create a `Resource` instance "from" a monitored resource descriptor; you just create one. The monitored resource descriptors are just machine-readable documentation, of a sort.
- You list monitored resource descriptors with the `Client.list_resource_descriptors` method, not "using the `ResourceDescriptor` class".
- In any case, the list of built-in monitored resource descriptors is not very useful in this context, because only a few monitored resource types make sense as a sender of custom metrics. Hence the text I suggested earlier.